test/bats/akeyless.bats

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bats
+
+ load helpers
+
+ BATS_TESTS_DIR=test/bats/tests/akeyless
+ WAIT_TIME=120
+ SLEEP_TIME=1
+
+
+ export LABEL_VALUE=${LABEL_VALUE:-"test"}
+ export ANNOTATION_VALUE=${ANNOTATION_VALUE:-"app=test"}
+
+@test "install akeyless csi provider" {
+   # create the ns akeyless
+   kubectl create ns akeyless
+   # install the akeyless provider using the helm charts
+
+   helm repo add akeyless https://akeylesslabs.github.io/helm-charts
+   helm repo update
+   helm install akeyless akeyless/akeyless-csi-provider --namespace=akeyless --set csi.image.pullPolicy=Always
+
+   # wait for akeyless and akeyless-csi-provider pods to be running
+   kubectl wait --for=condition=Ready --timeout=${WAIT_TIME}s pods --all -n akeyless
+ }
+
+@test "create akeyless k8s secret" {
+  run kubectl create secret generic secrets-store-creds --from-literal akeylessAccessKey=${AKEYLESS_ACCESS_KEY}
+  assert_success
+
+  # label the node publish secret ref secret
+  run kubectl label secret secrets-store-creds secrets-store.csi.k8s.io/used=true
+  assert_success
+}
+
+@test "secretproviderclasses crd is established" {
+   kubectl wait --for condition=established --timeout=${WAIT_TIME}s crd/secretproviderclasses.secrets-store.csi.x-k8s.io
+
+   run kubectl get crd/secretproviderclasses.secrets-store.csi.x-k8s.io
+   assert_success
+}
+
+@test "deploy akeyless secretproviderclass crd" {
+   kubectl apply -f $BATS_TESTS_DIR/secretproviderclass.yaml
+   kubectl wait --for condition=established --timeout=${WAIT_TIME}s crd/secretproviderclasses.secrets-store.csi.x-k8s.io
```

Comment:
This is already checked [here](https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/910/files#diff-882da5c47eff475fd29efc19d199866354b582b7e713fce8586fccb8911d5f05R36), so we can remove this line

Reply:
fixed